jkube-kit/enricher/api/src/main/java/org/eclipse/jkube/kit/enricher/api/util/KubernetesResourceUtil.java

```diff
@@ -702,6 +704,17 @@ public static void setSourceUrlAnnotationIfNotSet(HasMetadata item, String sourc
         });
     }
 
+    public static void removeResourceSourceUrlAnnotation(KubernetesListBuilder builder) {
+        builder.accept(new TypedVisitor<ObjectMetaBuilder>() {
+            @Override
+            public void visit(ObjectMetaBuilder objectMetaBuilder) {
+                if (objectMetaBuilder != null) {
+                    objectMetaBuilder.removeFromAnnotations(Constants.RESOURCE_SOURCE_URL_ANNOTATION);
+                }
+            }
+        });
+    }
```

Comment:
I'm still not sure why we still keep this and the related call from DependencyEnricher.

I understand that this might come from a Dependency JAR, but if the dependency has an aligned version of JKube, then it shouldn't be necessary.

IMHO it might be fine to keep this, but we should open a Deprecation issue for 2.x milestone, and add a `@deprecated` note+annotation both in this method with a link to the issue. WDYTH?

Reply:
I checked and it looks like this annotation was already marked as deprecated in https://github.com/eclipse/jkube/pull/1245 :

https://github.com/eclipse/jkube/blob/5773ee44818f862bb543e1a27a01628b259d56d0/jkube-kit/enricher/api/src/main/java/org/eclipse/jkube/kit/enricher/api/util/Constants.java#L24-L25

Reply:
Not the annotation, **the new method** should be marked as deprecated, since we're just adding it temporarily (right?).

Also create the issue to remove **this method** in 2.0 (plus add it as a comment in the code).

Reply:
I've created https://github.com/eclipse/jkube/issues/1465